src/components/Dropdown/Dropdown.tsx

```diff
@@ -247,23 +257,50 @@ export default class Dropdown extends AutoControlledComponent<
                   isOpen,
                   highlightedIndex,
                 )}
-              </div>
-            )
-          }}
-        </Downshift>
-      </ElementType>
+              </ElementType>
+            </Ref>
+          )
+        }}
+      </Downshift>
+    )
+  }
+
+  private renderTriggerButton(
+    getToggleButtonProps: (options?: GetToggleButtonPropsOptions) => any,
+    styles: ComponentSlotStylesInput,
+  ): JSX.Element {
+    const { placeholder, itemToString } = this.props
+    const { value } = this.state
+    return (
+      <Button
+        content={value ? itemToString(value) : placeholder}
```

Comment:
I would expect the user to be able to provide some shorthand for this button, maybe even different shorthands for opened and closed toggle button. This, however can be the default button if nothing is provided (I liked the styling and the look of it 👍 )

Reply:
agree that we would like to have this aspect of dropdown being customizable at the end (not concern of this PR, however)

Reply:
yes, I agree, I just wanted to have this as a place to start. Added it to https://github.com/stardust-ui/react/issues/560 for tracking.